Gruntfile.circle.js

```diff
@@ -198,6 +199,18 @@ module.exports = function gruntConfig(grunt) {
 
     shell: {},
 
+    stylelint: {
+      options: {
+        configFile: `.stylelintrc`,
+        format: `css`
+      },
+      src: [
+        `./packages/**/*.css`,
+        `!./packages/*/node_modules/**/*.css`,
+        `!./packages/*/dist/**/*.css`
+      ]
+    },
+
```

Comment:
Running the specific package takes very little time, but running through circle takes forever. Is there anything i should be doing here @ianwremmel ?